.circleci/config.yml

```diff
@@ -16,6 +16,17 @@ jobs:
           name: "test"
           command: |
               ./test.sh
+      - run:
+          name: Build Docker images
+          command: |
+              ./build-images.sh 11
```

Comment:
Instead of passing magic numbers to the scripts, can we create variables with meaningful names(nodeVersion11 or similar ) and use those instead?

Reply:
If you check inside the **build-images.sh**, the variables are assigned to meaningful names